backtrace-supported.h.cmake

```diff
@@ -0,0 +1,51 @@
+/* backtrace-supported.h.cmake -- Whether stack backtrace is supported.

```

Comment:
Is there some way we could generate this file from backtrace-supported.h.in, so that we don't have the unnecessary duplication?

Reply:
I've checked right now and it's entirely possible. I'll do proper changes right away. config.h.in uses autotools-specific syntax, so unfortunately we need CMake variant.